src/provider/provider_file_memory.c

```diff
@@ -101,6 +105,10 @@ file_translate_params(umf_file_memory_provider_params_t *in_params,
         return result;
     }
 
+    /// IPC is enabled only for UMF_MEM_MAP_SHARED or UMF_MEM_MAP_SYNC visibility
```

Comment:
I guess this comment is redundant - it doubles the simple information from the code below, I believe

Reply:
Done